### PR TITLE
Farcaster Platform Integration

### PR DESCRIPTION
**Pull request by rbennettcw**
## Link to Issue
Closes: #8773 
Closes: #8774
Closes: #8775
Closes: #8983 

## Description of Changes
- Adds back the farcaster frames
- Adds farcaster metadata to ContestManager
- Adds farcaster webhooks + upvote action
- Adds FarcasterWorkerPolicy + RMQ configs for farcaster events

## Test Plan
- See docs for setting up local env + testing instructions: https://github.com/hicommonwealth/commonwealth/blob/ryan/farcaster-platform/common_knowledge/Farcaster-Contests.md

## Deployment Plan
- Add envs `NEYNAR_API_KEY`, `NEYNAR_REPLY_WEBHOOK_URL`, `FARCASTER_ACTION_URL`

## Other Considerations
N/A

### CODE REVIEW

File: libs/model/src/contest/CreateContestManagerMetadata.command.ts [hunk 1]
```diff
@@ -44,6 +44,8 @@ export function CreateContestManagerMetadata(): Command<
         }));
       }
 
+      const farcaster_frame_url = `/api/farcaster/${payload.contest_address}`;
```
Comment (masvelio) [inline libs/model/src/contest/CreateContestManagerMetadata.command.ts:47]: would this lead to our api? like `localhost:3000/api/farcaster/0x123`? 
asking because we have already router hooked up to `api/farcaster` [endpoint](https://github.com/hicommonwealth/commonwealth/blob/4d668b485f6b8dbdbb1b5f04d8505202782c530f/packages/commonwealth/server/routing/router.ts#L1128) that handles farcaster frames

Comment (rbennettcw) [reply libs/model/src/contest/CreateContestManagerMetadata.command.ts:47]: Yes, that URL will point to the farcaster frame endpoint and be rendered to the frontend so that the user can copy/paste it. We're using your existing endpoint for the target.

File: libs/model/src/policies/utils.ts [hunk 2]
```diff
@@ -0,0 +1,189 @@
+import { Actor, logger } from '@hicommonwealth/core';
```
Comment (Rotorsoft) [inline libs/model/src/policies/utils.ts:1]: just a naming question... are these policy utils or contest utils? ... let's think about how to name reusable command functions

**Review by Rotorsoft — APPROVED** [review]
Tested by following the steps in the documentation and got everything to work.

- Left a note about naming utils that can be addresses here or later
- Left some comments about how to add more details to the documentation
- We need to test all the edge cases, and the UI should be intuitive ... this is a complex flow with multiple on-chain transactions

**Review by masvelio — APPROVED** [review]
tested, lgtm!
